### DISCUSSION TITLE
No notification after initial start up

### DISCUSSION THREAD

**Discussion by robinhood-code** [containrrr/watchtower#1826] [Q&A] [answered]
Hi,

I have setup notification for watchtower. It worked for the start up (got the startup notification in Discord on 6 Nov), but I got no subsequent notification. In fact the log is showing "notify=no". Is there any parameter to change to "notify=yes" ? 

I'm running notify as a container in a Synology NAS. Thanks. 

2023/11/09 22:00:10 | stderr | time="2023-11-09T22:00:10+11:00" level=info msg="Session done" Failed=0 Scanned=4 Updated=0 notify=no
-- | -- | --
2023/11/08 22:00:09 | stderr | time="2023-11-08T22:00:09+11:00" level=info msg="Session done" Failed=0 Scanned=4 Updated=0 notify=no
2023/11/07 22:00:09 | stderr | time="2023-11-07T22:00:09+11:00" level=info msg="Session done" Failed=0 Scanned=4 Updated=0 notify=no
2023/11/06 22:00:11 | stderr | time="2023-11-06T22:00:11+11:00" level=info msg="Session done" Failed=0 Scanned=4 Updated=0 notify=no
2023/11/06 14:57:48 | stderr | time="2023-11-06T14:57:48+11:00" level=info msg="Note that the first check will be performed in 7 hours, 2 minutes, 11 seconds"
2023/11/06 14:57:48 | stderr | time="2023-11-06T14:57:48+11:00" level=info msg="Scheduling first run: 2023-11-06 22:00:00 +1100 AEDT"
2023/11/06 14:57:48 | stderr | time="2023-11-06T14:57:48+11:00" level=info msg="Checking all containers (except explicitly disabled with label)"
2023/11/06 14:57:48 | stderr | time="2023-11-06T14:57:48+11:00" level=info msg="Using notifications: discord"
2023/11/06 14:57:48 | stderr | time="2023-11-06T14:57:48+11:00" level=info msg="Watchtower 1.6.0"

![image](https://github.com/containrrr/watchtower/assets/6055272/8bf27cee-782b-4ab6-8ad1-1513c4306168)

**Comment by piksel** [comment, marked as answer]
The default is to only send notifications if there were any updates or errors. The log messages with `notify=no` tags are just used to allow logging information without sending notifications. If you want the notifications to behave differently you can customize the template, see: https://containrrr.dev/watchtower/notifications/#report_templates
